app/models/package_manager/puppet.rb

```diff
@@ -0,0 +1,41 @@
+module PackageManager
+  class Puppet < Base
+    HAS_VERSIONS = false
+    HAS_DEPENDENCIES = false
+    BIBLIOTHECARY_SUPPORT = false
+    BIBLIOTHECARY_PLANNED = true
+    URL = 'https://forge.puppet.com'
+    COLOR = '#302B6D'
+
+    def self.project_names
+      offset = 0
+      projects = []
+      while true
+        results = get_json("https://forgeapi.puppetlabs.com/v3/modules?limit=100&offset=#{offset}")['results'].map { |result| result['slug'] }
+	break if results == []
+	projects += results
+	offset +=100
+      end
+      projects
+    end
+
+    def self.project(name)
+      get_json("https://forgeapi.puppetlabs.com/v3/modules/#{name}")
+    end
+
+    def self.mapping(project)
+      {
+        name: project['slug'],
+        homepage: "https://forge.puppet.com/#{project['slug']}",
```

Comment:
This should be the the `project_page` attribute, we can generate the link to the forge page from the name by adding a `package_link` method like: https://github.com/librariesio/libraries.io/blob/e38e012b4b9dd49e0b20ff242279d37edd5940ed/app/models/package_manager/rubygems.rb#L10-L12